Section3/Exercise7/Main7.cpp

```diff
@@ -6,11 +6,11 @@ For each number that is matched in the same index, give the user $10.*/
 #include <ctime>  
 using namespace std;
 bool repeatValues(int arrRepeat[], int size) {
-	for (int pos = 1; pos < size; pos++)
+	for (int Loteryposition = 1; Loteryposition < size; Loteryposition++)
```

Comment:
Two comments here:
- Lottery is written with double 't', there's a typo here.
- Prevent coupling your functions from their specific use. This method is supposed to check for repeating numbers in an array, which has nothing to do with a Lottery. With that in mind, your variables inside the function shouldn't be named as if they were related to the Lottery.

Reply:
i, j, and k are universally accepted variables for loop iterations and are appropriate for a simple algorithm such as this one. That's an acceptable exception to the naming conventions :)